[docs] improve descriptions for our APIs, and add videos (#6570)
expo/expo: docs/pages/versions/unversioned/sdk/video-thumbnails.md

---
title: VideoThumbnails
sourceCodeUrl: 'https://github.com/expo/expo/tree/sdk-36/packages/expo-video-thumbnails'
---

> **`expo-video-thumbnails`** allows you to generate an image to serve as a thumbnail for a provided video.

Reviewer: ```suggestion
**`expo-video-thumbnails`** allows you to generate an image to serve as a thumbnail from a video file.
```
